Formula/topgrade.rb

```diff
@@ -0,0 +1,17 @@
+class Topgrade < Formula
+  desc "Upgrade all the things"
+  homepage "https://github.com/r-darwish/topgrade"
+  url "https://github.com/r-darwish/topgrade/archive/v0.13.0.tar.gz"
+  sha256 "c1a7fe9675a4572f77afad21849ce5e512c194e4e00de17ce5590f5588ad38b9"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix
+  end
+
+  test do
+    output = shell_output("#{bin}/topgrade -n")
+    assert_match "Dry running: /usr/local/bin/brew upgrade", output
```

Comment:
May wish to use `#{HOMEBREW_PREFIX}/bin/brew` rather than assuming `/usr/local`.

Reply:
Done